examples/overlay/messages/10_output_jaccard.json

```diff
@@ -0,0 +1,122 @@
+{
+    "query_graph": {
+        "nodes": {
+            "type-2 diabetes": {
+                "id": "MONDO:0005148"
+            },
+            "protein": {
+                "category": "biolink:Protein"
+            },
+            "drug": {
+                "category": "biolink:ChemicalSubstance"
+            }
+        },
+        "edges": {
+            "correlated": {
+                "subject": "protein",
+                "predicate": "biolink:correlated_with",
+                "object": "type-2 diabetes"
+            },
+            "interacts": {
+                "subject": "drug",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "protein"
+            },
+            "j1": {
+                "subject": "drug",
+                "predicate": "biolink:has_jaccard_index_with",
+                "object": "type-2 diabetes"
+            }
+        }
+    },
+    "knowledge_graph": {
+        "nodes": {
+            "MONDO:0005148": {"name": "type-2 diabetes"},
+            "CHEBI:6801": {
+                "name": "metformin",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "CHEBI:5441": {
+                "name": "glyburide",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "UniProtKB:Q92786": {
+                "name": "PROX1",
+                "category": "biolink:Protein"
+            },
+            "UniProtKB:P30279": {
+                "name": "CCND2",
+                "category": "biolink:Protein"
+            }
+        },
+        "edges": {
+            "k7g89spc": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:P30279"
+            },
+            "df87ff82": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:Q92786"
+            },
+            "5133c100": {
+                "subject": "CHEBI:5441",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:P30279"
+            },
+            "vk456hb1": {
+                "subject": "UniProtKB:Q92786",
+                "predicate": "biolink:correlated_with",
+                "object": "MONDO:0005148"
+            },
+            "lkg89002": {
+                "subject": "UniProtKB:P30279",
+                "predicate": "biolink:correlated_with",
+                "object": "MONDO:0005148"
+            },
+            "j1_1": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:has_jaccard_index_with",
+                "object": "MONDO:0005148",
+                "attributes": [
+                    {
+                      "name": "jaccard_index",
+                      "value": "0.66",
```

Comment:
I'm a bit confused about this computation. The drug and the disease are each connected to two proteins, and they are the same two. Should the Jaccard index not be 1.0?

Reply:
Oh yeah I see what you mean. I must have made a mistake here. I think I was originally going to put 3 different proteins but went with 2. I can fix it to be 1.0 and .5 instead of .66 and .33.

Reply:
So after reading the next comment down would it be better to change the values to 1.0 & 0.5 or instead add in a 3rd protein node and keep the values the same?

Reply:
So I've changed the values to 1.0 & 0.5 instead of adding in a 3rd protein node. LMK if you would rather I add in the node instead.